xml/docu_styleguide.language.xml

```diff
@@ -35,9 +35,8 @@
  <sect2 xml:id="sec-abbreviation">
   <title>Abbreviations</title>
   <para>
-   Where possible, avoid using abbreviations. Especially avoid unusual
-   abbreviations. You may create plurals of acronyms. In all other cases,
-   avoid creating plurals of abbreviations.
+   Where possible, avoid using abbreviations, especially unusual
+   ones. Avoid using plurals of abbreviations, except for acronyms.
```

Comment:
To be honest, I find the old version clearer, especially with respect to the sentence(s) about plurals.

Reply:
I would also specifically call out Latin abbreviations, too. Even people who wouldn't use "repo" will go right ahead and use "eg" :woman_shrugging: 

Also, pedantry: you might want to say "acronyms and initialisms"

Reply:
You are right about Latin abbreviations needing to be called out--but this section already has a subsection "Latin Abbreviations" (see ~25l below).

Reply:
Ah, my bad.

Reply:
@Loquacity: On which line exactly are you suggesting to use "acronyms and initialisms"?